src/sentry/static/sentry/app/views/performance/transactionDetails/utils.tsx

```diff
@@ -72,6 +73,11 @@ export function generateTraceTarget(
   organization: OrganizationSummary
 ): LocationDescriptor {
   const traceId = event.contexts?.trace?.trace_id ?? '';
+
+  if (organization.features.includes('trace-view-summary')) {
```

Comment:
Should make sure we remove Sentry from this flag (and add Visibility) once this lands